radicle-cli/src/commands/unassign.rs

```diff
@@ -1,31 +1,30 @@
 use std::ffi::OsString;
-use std::str::FromStr;
 
 use anyhow::anyhow;
 use nonempty::NonEmpty;
-use radicle::prelude::Did;
 
-use crate::terminal as term;
-use crate::terminal::args::{Args, Error, Help};
 use radicle::cob;
 use radicle::cob::issue;
+use radicle::prelude::Did;
 use radicle::storage::WriteStorage;
 
+use crate::terminal as term;
+use crate::terminal::args::{Args, Error, Help};
+
 pub const HELP: Help = Help {
     name: "unassign",
     description: "Unassign an issue",
     version: env!("CARGO_PKG_VERSION"),
     usage: r#"
 Usage
 
-    rad unassign <issue> --from <did>
+    rad unassign <issue-id> --from <did> [<option>...]
 
     To unassign multiple users from an issue, you may repeat
     the `--from` option.
 
 Options
 
-    --from <did>     Assignee to remove from the issue
```

Comment:
I'd keep this somewhere.

Reply:
kept it above `Options` https://github.com/radicle-dev/heartwood/pull/445/commits/6d0b338c6920de263d264d7bac5e5c819bc80511#diff-adeb3a18621bbf65c7c730ffe078573080b9ae6277929f888af42cd45301ecd2R26